roles/openshift/defaults/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+# Common variables
+openshift_env: 'test'
+openshift_roles: [ 'master', 'node' ]
+openshift_bind_ip: "{{ ansible_default_ipv4.address }}"
+openshift_public_ip: "{{ ansible_default_ipv4.address }}"
```

Comment:
One thing that I am hoping to do as well, is to be able to run the openshift role on a non-provisioned stand alone host (local vm, etc) and without having to configure anything apply the role and get a all-in-one origin instance.